### PR TITLE
Add available S3 backup videos to videos page carousels

### PR DESCRIPTION
**Pull request by kelbyhawn**
Adds backups videos from S3 to https://code.org/videos. I couldn't find backups for all the videos, but was able to add some to the following sections:
- Blockchain
- Careers in CS
- Inspirational Videos

The videos without the YouTube play button are the backups. 

**Jira ticket:** [ACQ-1802](https://codedotorg.atlassian.net/browse/ACQ-1802)

### CODE REVIEW

File: pegasus/sites.v3/code.org/views/carousel_careers_in_cs_video.haml [hunk 1]
```diff
@@ -1,6 +1,8 @@
...
+- video_careers_caption = [hoc_s(:video_library_page_cs_video_title_1), hoc_s(:video_library_page_cs_video_title_6), hoc_s(:video_library_page_cs_video_title_2), hoc_s(:video_library_page_cs_video_title_7), hoc_s(:video_library_page_cs_video_title_3), hoc_s(:video_library_page_cs_video_title_8), hoc_s(:video_library_page_cs_video_title_4), hoc_s(:video_library_page_cs_video_title_9), hoc_s(:video_library_page_cs_video_title_5)]
+
+- video_careers_download = ["", "", "", "", "//videos.code.org/careers-in-tech/my_name_is_brina.mp4", "//videos.code.org/careers-in-tech/my_name_is_federico.mp4", "//videos.code.org/careers-in-tech/my_name_is_kinsley.mp4", "//videos.code.org/careers-in-tech/my_name_is_tess.mp4", "//videos.code.org/careers-in-tech/my_name_is_polina.mp4"]
```
Comment (kelbyhawn) [inline pegasus/sites.v3/code.org/views/carousel_careers_in_cs_video.haml:5]: https://github.com/code-dot-org/code-dot-org/assets/9256643/9b92e78b-1aba-44d4-b189-138c7293d600

File: pegasus/sites.v3/code.org/views/carousel_inspirational_video.haml [hunk 2]
```diff
@@ -2,14 +2,16 @@
...
 
+- video_inspirational_download = ["//videos.code.org/social/change_the_face_of_cs.mp4", "//videos.code.org/social/athlete-steph-curry.mp4", "//videos.code.org/social/athletes-hour-of-code.mp4", "//videos.code.org/social/cs-is-everything.mp4", "//videos.code.org/social/ChangeTheWorld_v5.mp4", "", "//videos.code.org/social/what-most-schools-dont-teach.mp4", "//videos.code.org/2015/social/president_obama_asks_america.mp4", "", "//videos.code.org/social/codeorg-60sec.mp4", "//videos.code.org/social/my-name-is-aracely.mp4", "//videos.code.org/social/latam-hour-of-code-2018.mp4", "", "", "", "//videos.code.org/ywics-panel-2023.mp4"]
```
Comment (kelbyhawn) [inline pegasus/sites.v3/code.org/views/carousel_inspirational_video.haml:5]: https://github.com/code-dot-org/code-dot-org/assets/9256643/6b10f0c5-9d8d-4df6-b24a-4c67c8a22f2b

File: pegasus/sites.v3/code.org/views/carousel_blockchain_video.haml [hunk 3]
```diff
@@ -2,14 +2,16 @@
 
 - video_blockchain_caption = [hoc_s(:blockchain_video_title_01), hoc_s(:blockchain_video_title_02), hoc_s(:blockchain_video_title_03), hoc_s(:blockchain_video_title_04), hoc_s(:blockchain_video_title_05), hoc_s(:blockchain_video_title_06), hoc_s(:blockchain_video_title_07)]
 
+- video_blockchain_download = ["//videos.code.org/blockchain/Blockchain_1_Intro.mp4", "//videos.code.org/blockchain/Blockchain_2_Why_Blockchain.mp4", "//videos.code.org/blockchain/Blockchain_3_Under_the_Hood.mp4", "//videos.code.org/blockchain/Blockchain_5_Marketplaces.mp4", "//videos.code.org/blockchain/Blockchain_5_Marketplaces.mp4", "//videos.code.org/blockchain/Blockchain_6_Societal_Impacts_01.mp4", "//videos.code.org/blockchain/Blockchain_7_Societal_Impacts_02.mp4"]
```
Comment (kelbyhawn) [inline pegasus/sites.v3/code.org/views/carousel_blockchain_video.haml:5]: https://github.com/code-dot-org/code-dot-org/assets/9256643/4f20969b-0109-4376-8a4d-b4d1c2f636d2

File: pegasus/sites.v3/code.org/views/carousel_careers_in_cs_video.haml [hunk 4]
```diff
@@ -1,6 +1,8 @@
 - video_careers_index = 9
 
-- video_careers_caption = [hoc_s(:video_library_page_cs_video_title_1), hoc_s(:video_library_page_cs_video_title_6), hoc_s(:video_library_page_cs_video_title_2), hoc_s(:video_library_page_cs_video_title_7), hoc_s(:video_library_page_cs_video_title_3), hoc_s(:video_library_page_cs_video_title_8), hoc_s(:video_library_page_cs_video_title_4), hoc_s(:video_library_page_cs_video_title_8), hoc_s(:video_library_page_cs_video_title_5), hoc_s(:video_library_page_cs_video_title_9)]
+- video_careers_caption = [hoc_s(:video_library_page_cs_video_title_1), hoc_s(:video_library_page_cs_video_title_6), hoc_s(:video_library_page_cs_video_title_2), hoc_s(:video_library_page_cs_video_title_7), hoc_s(:video_library_page_cs_video_title_3), hoc_s(:video_library_page_cs_video_title_8), hoc_s(:video_library_page_cs_video_title_4), hoc_s(:video_library_page_cs_video_title_9), hoc_s(:video_library_page_cs_video_title_5)]
```
Comment (kelbyhawn) [inline pegasus/sites.v3/code.org/views/carousel_careers_in_cs_video.haml:3]: Fixed a mis-matched caption here.

**Review by bethanyaconnor — APPROVED** [review]

**Pull request merged**